x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/crawl_requests_table.test.tsx

```diff
@@ -12,26 +12,27 @@ import React from 'react';
 
 import { shallow, ShallowWrapper } from 'enzyme';
 
-import { EuiBasicTable, EuiEmptyPrompt } from '@elastic/eui';
+import { EuiBadge, EuiBasicTable, EuiEmptyPrompt } from '@elastic/eui';
 
 import { mountWithIntl } from '../../../../test_helpers';
 
-import { CrawlEvent, CrawlerStatus } from '../types';
+import { CrawlEvent, CrawlerStatus, CrawlType } from '../types';
 
-import { CrawlRequestsTable } from './crawl_requests_table';
+import { CrawlRequestsTable, CrawlEventTypeBadge } from './crawl_requests_table';
+
+const MOCK_EVENT: CrawlEvent = {
```

Comment:
Thanks for typing this. At some point in this plugin's development it became common practice not to type mock values in tests, but I'm glad we're getting away from that.

Reply:
Yeah I think its super useful (also makes it easier to check which test files I need to update when I update a type)